source/lib-intro.tex

```diff
@@ -476,7 +476,7 @@
 appropriate):\footnote{To save space, items that do not apply to a function are omitted.
 For example, if a function does not specify any
 further
-preconditions, there will be no ``Requires'' paragraph.}
+preconditions, there will be no \requires paragraph.}
```

Comment:
Hm, we should probably also (but separately) decide one only one of paragraph, section or clause.